src/types/application.types.ts

```diff
@@ -14,6 +19,6 @@ export interface ListedApplication {
 
 export interface ApplicationDataset {
   id: number;
-  title: string;
-  url: string;
+  title: Label[];
+  url: Label[];
```

Comment:
question: is URL also a label?

Reply:
yes it seems
https://github.com/GenomicDataInfrastructure/gdi-userportal-access-management-service/blob/main/src/main/openapi/daam.yaml#L747